### PR TITLE
upload translation for chapter 13

### PR DESCRIPTION
**Pull request by Zheng7426**
Hi @yukie1412, please kindly review it when you have time. Thank you.

### CODE REVIEW

File: 13_browser.md [hunk 1]
```diff
@@ -1,458 +1,263 @@
...
+
+这既是一个祝福，也是一个咒诅。一方面，由于没有一个中央组织对系统进行控制，导致多个组织之间通过松散的((合作))(偶尔还有公开互怼的情况)来对其进行提升。另一方面，万维网以这种无组织、无纪律的方式被开发，意味着最终的系统并不是一个拥有内在((一致性))的光辉范例。它有些部分是比较令人困惑的，而且构思拙劣。
+
+## 网络与互联网
+
+计算机((网络))自1950年代就出现了，当你在两台或两台以上的计算机之间连上缆线，并且允许它们通过这些缆线来回发送数据的时候，你便可以做很多神奇的事情。
+
+如果连接同一栋楼里的两台机器可以让我们做神奇的事情，那么将遍布地球上的机器都连接上的话应该会更棒。开始实现这个愿景的技术在 1980 年代被开发，最终产生的网络被称为 _((互联网))_。互联网没有辜负人们对它的期许。
+
+一台计算机可以使用这个网络来向另外一台计算机发送比特(bits)。为了使这发送比特的方式产生有效的((通信))，在两端的计算机必须知道这些比特表达的是什么。一个给定序列的比特的含义完全取决于该序列表达的信息类型和使用的((编码))机制。
```
Comment (ghost) [inline 13_browser.md]: are you sure the translation for 'bits' is 比特? google translation says it's 位

File: 13_browser.md [hunk 2]
```diff
@@ -1,458 +1,263 @@
...
-
-Machines connected to the Internet get an _((IP address))_, which is a
-number that can be used to send messages to that machine, and looks
-something like `149.210.142.219` or `2001:4860:4860::8888`. But lists
-of more or less random numbers are hard to remember and awkward to
-type, so you can instead register a _((domain)) name_ for a specific
-address or set of addresses. I registered _eloquentjavascript.net_ to
-point at the IP address of a machine I control and can thus use that
-domain name to serve web pages.
+第一个部分告诉我们这个 URL 使用 HTTP ((协议)) (比如说如果是加密的 HTTP，那会是 _https://_)。中间的部分指出我们从哪一个 ((服务器)) 请求文件。最后一个部分则是我们感兴趣的具体文件((或 _((资源))_))的路径字符串。
```
Comment (ghost) [inline 13_browser.md]: > 比如说如果是加密的 HTTP，那会是 _https://_

i would say "而不是加密的 HTTP，即 _https://_"

File: 13_browser.md [hunk 3]
```diff
@@ -1,458 +1,263 @@
...
 {{index [escaping, "in HTML"]}}
 
-To be able to include ((angle brackets)) in the text of a document,
-even though they have a special meaning in HTML, yet another form of
-special notation has to be introduced. A plain opening angle bracket
-is written as `&lt;` ("less than"), and a closing bracket is written
-as `&gt;` ("greater than"). In HTML, an ampersand (`&`) character
-followed by a name or character code and a semicolon (`;`) is called an _((entity))_
-and will be replaced by the character it encodes.
+为了能够在文档的文本里包含((角括号))，由于它们在 HTML 中有特殊含义，需引入另一种特殊的写法。一个普通的起始角括号写成 `&lt;`("less than")，一个闭合角括号写成 `&gt;` ("greater than")。在 HTML 中，`&` 字符后跟一个名称或字符码、一个分号 (`;`) —— 这样的组合被称为一个 _((实体))_，并且会被其编码的字符给替换掉。
```
Comment (ghost) [inline 13_browser.md]: you should translate "less than" & "greater than" as well

File: 13_browser.md [hunk 4]
```diff
@@ -1,458 +1,263 @@
...
 {{index "head (HTML tag)", "body (HTML tag)", "title (HTML tag)", "h1 (HTML tag)", "p (HTML tag)"}}
 
-HTML documents have a head and a body. The head contains information
-_about_ the document, and the body contains the document itself. In
-this case, the head declares that the title of this document is "My
-home page" and that it uses the UTF-8 encoding, which is a way to
-encode Unicode text as binary data. The document's body contains a
-heading (`<h1>`, meaning "heading 1"—`<h2>` to `<h6>` produce
-subheadings) and two ((paragraph))s (`<p>`).
+HTML 文档有一个头部和主体。头部包含了 _关于_ 文档的信息，而主体包含文档本身。在上述例子中，头部声明了该文档的抬头是“我的主页”，并使用 UTF-8 编码 —— 将 Unicode 文本编码为二进制数据的一种方式。该文档的主体包含了一个标题(`<h1>`，代表“一级标题” —— `<h2>` 至 `<h6>` 可产生不同大小的子标题)以及两个((段落))的 (`<p>`)。
```
Comment (ghost) [inline 13_browser.md]: i think the final "的" is unnecessary

File: 13_browser.md [hunk 5]
```diff
@@ -1,458 +1,263 @@
...
-
-This was the dark age of ((compatibility)), often called the
-_((browser wars))_. Web developers were left with not one unified Web
-but two or three incompatible platforms. To make things worse, the
-browsers in use around 2003 were all full of ((bug))s, and of course
-the bugs were different for each ((browser)). Life was hard for people
-writing web pages.
+在万维网的早期，一个名为 ((Mosaic)) 的浏览器统治着市场。过了短短几年之后，天平移向了((Netscape))，而它随后又被微软的((Internet Explorer))占据了大部分市场。无论何时，每当有一个((浏览器))统治了市场，该浏览器厂商便会想要独自为万维网发明新的特性。由于大多数用户使用最流行的浏览器，((网站))便会开始使用这些特性 —— 而不管别的浏览器是否兼容。
+
+这是((兼容性))的黑暗时代，通常被称为 _((浏览器大战))_。网页开发者并没有一个统一的万维网，摆在他们面前的是两到三个互不兼容的平台。更糟糕的是，2003 年左右的浏览器有很多 ((bug))，当然每个((浏览器))的 bug 还都不一样。那时候网页开发者的生活糟透了。
```
Comment (ghost) [inline 13_browser.md]: bug should be pluralized

File: 13_browser.md (hunk 1 above)
Comment (Zheng7426) [reply 13_browser.md]: 位 is better, you are right.

File: 13_browser.md (hunk 2 above)
Comment (Zheng7426) [reply 13_browser.md]: Agreed.

File: 13_browser.md (hunk 3 above)
Comment (Zheng7426) [reply 13_browser.md]: Will do.

File: 13_browser.md (hunk 4 above)
Comment (Zheng7426) [reply 13_browser.md]: Will delete it.

File: 13_browser.md (hunk 5 above)
Comment (Zheng7426) [reply 13_browser.md]: No problem. Btw, It is good to see you here on Github again :)

**Review by ghost — APPROVED** [review]

**Pull request merged**